{
  "dataType": "CVE_RECORD",
  "dataVersion": "5.1",
  "cveMetadata": {
    "cveId": "CVE-2024-36985",
    "assignerOrgId": "42b59230-ec95-491e-8425-5a5befa1a469",
    "state": "PUBLISHED",
    "assignerShortName": "Splunk",
    "dateReserved": "2024-05-30T16:36:20.999Z",
    "datePublished": "2024-07-01T16:30:57.461Z",
    "dateUpdated": "2024-07-01T16:57:39.776Z"
  },
  "containers": {
    "cna": {
      "affected": [
        {
          "product": "Splunk Enterprise",
          "vendor": "Splunk",
          "versions": [
            {
              "version": "9.2",
              "status": "affected",
              "versionType": "custom",
              "lessThan": "9.2.2"
            },
            {
              "version": "9.1",
              "status": "affected",
              "versionType": "custom",
              "lessThan": "9.1.5"
            },
            {
              "version": "9.0",
              "status": "affected",
              "versionType": "custom",
              "lessThan": "9.0.10"
            }
          ]
        }
      ],
      "descriptions": [
        {
          "lang": "en",
          "supportingMedia": [
            {
              "base64": false,
              "type": "text/html",
              "value": "In Splunk Enterprise versions below 9.2.2, 9.1.5, and 9.0.10, a low-privileged user that does not hold the admin or power Splunk roles could cause a Remote Code Execution through an external lookup that references the “splunk_archiver“ application."
            }
          ],
          "value": "In Splunk Enterprise versions below 9.2.2, 9.1.5, and 9.0.10, a low-privileged user that does not hold the admin or power Splunk roles could cause a Remote Code Execution through an external lookup that references the “splunk_archiver“ application."
        }
      ],
      "references": [
        {
          "url": "https://advisory.splunk.com/advisories/SVD-2024-0705"
        },
        {
          "url": "https://research.splunk.com/application/8598f9de-bba8-42a4-8ef0-12e1adda4131"
        }
      ],
      "title": "Remote Code Execution (RCE) through an external lookup due to “copybuckets.py“ script in the “splunk_archiver“ application in Splunk Enterprise",
      "datePublic": "2024-07-01T00:00:00.000000",
      "metrics": [
        {
          "cvssV3_1": {
            "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:H/A:H",
            "version": "3.1",
            "baseScore": 8.8,
            "baseSeverity": "HIGH"
          },
          "format": "CVSS",
          "scenarios": [
            {
              "lang": "en",
              "value": "GENERAL"
            }
          ]
        }
      ],
      "problemTypes": [
        {
          "descriptions": [
            {
              "lang": "en",
              "type": "cvw",
              "description": "The software calls a function, procedure, or routine, but the caller specifies an argument that contains the wrong value, which may lead to resultant weaknesses.",
              "cweId": "CWE-687"
            }
          ]
        }
      ],
      "source": {
        "advisory": "SVD-2024-0705"
      },
      "credits": [
        {
          "lang": "en",
          "value": "Alex Hordijk"
        }
      ],
      "providerMetadata": {
        "orgId": "42b59230-ec95-491e-8425-5a5befa1a469",
        "shortName": "Splunk",
        "dateUpdated": "2024-07-01T16:57:39.776Z"
      }
    }
  }
}